ts/output/chtml/Wrappers/msubsup.ts

```diff
@@ -106,6 +106,7 @@ CommonMsubsupMixin<CHTMLWrapper<any, any, any>, Constructor<CHTMLscriptbase<any,
         const chtml = this.standardCHTMLnode(parent);
         const [base, sup, sub] = [this.baseChild, this.supChild, this.subChild];
         const [u, v, q] = this.getUVQ(base.getBBox(), sub.getBBox(), sup.getBBox());
+        const x = (this.baseCore.bbox.ic ? this.coreIC() * this.coreScale() : 0);
```

Comment:
No need for parentheses.

Reply:
We've had this discussion before, and agreed to allow them (I prefer them), but I've removed them for you.